### DISCUSSION TITLE
Request for update of the information of Tianshu Chen

### DISCUSSION THREAD

**Discussion by tianshu129** [trutzig89182/Mastodon-Sociologists#61] [General]
Would you please rewrite the information of Tianshu Chen in "https://jorgesat.github.io/Mastodon-Atmospheric/" according to the new information in the line 56 of Google Sheet "https://docs.google.com/spreadsheets/d/1qgj_XQMoyF7Prxz8m0R3MBuC-NdtrX3vHIj-kkVn6aU/edit?resourcekey=undefined#gid=644742588"? Many people followed the old ID of mine. Thank you.

**Comment by tianshu129** [comment]
Done.

**Reply by trutzig89182** [reply]
Please make sure you contact the actual maintainer of your specific list, with these issues. I won’t be able to change any specific information.

**Reply by tianshu129** [reply]
Thank you. Got it.
